swift/Workflow/Sources/SubtreeManager.swift

```diff
@@ -85,7 +90,7 @@ extension WorkflowNode {
             let queuedEvents = eventPipes.compactMap { pipe in
                 pipe.pendingOutput()
             }
-            if queuedEvents.count > 0 {
+            if !queuedEvents.isEmpty {
```

Comment:
not material to this review, but I’ve personally always had a harder time skimming `!thing.isEmpty` vs `thing.count > 0`

Reply:
I think this is mostly a defensive thing — `Collection.isEmpty` is O(1) where `Collection.count` _can_ be O(n). (Granted, we're using an `Array`, which conforms to `RandomAccessCollection` and so `count` is O(1), but 🤷.)